modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -1504,6 +1505,16 @@
 		end
 	end
 
+	function m.useFullPaths(cfg)
+		if cfg.useFullPaths ~= nil then
+			if cfg.useFullPaths then
+				m.element("UseFullPaths", nil, "true")
+			else
+				m.element("UseFullPaths", nil, "false")
+			end
```

Comment:
This could be replaced with:
```suggestion
			m.element("UseFullPaths", nil, iif(cfg.useFullPaths, "true", "false"))
```